provider/spark.go

```diff
@@ -47,12 +47,14 @@ const (
 	Materialize       JobType = "Materialization"
 	Transform         JobType = "Transformation"
 	CreateTrainingSet JobType = "Training Set"
+	BatchFeatures     JobType = "Batch Features"
 )
 
 const MATERIALIZATION_ID_SEGMENTS = 3
 const ENTITY_INDEX = 0
 const VALUE_INDEX = 1
 const TIMESTAMP_INDEX = 2
+const SPARK_SUBMIT_PARAMS_BYTE_LIMIT = 10_240
```

Comment:
Was this the limit or flat 10000?

Reply:
The docs say 10K bytes so I took that literally.